Jenkinsfile

```diff
@@ -271,10 +287,12 @@ node (node_label) {
 		}
 
 		// stage to run OCP 4.X scaleup
-		if (ocpv4_scale == "TRUE") {
-			env.WORKLOAD_PROPERTIES_FILE=OPENSHIFTv4_SCALE_PROPERTY_FILE
-			env.WORKLOAD="ATS-SCALE-CI-SCALE"
-			load "pipeline-scripts/workload.groovy"
+		if (ocpv4_scale == "TRUE") {	
+			if (openshiftv4_install_on_bm == "FALSE") {		
```

Comment:
Maybe skip the respective stage as a whole in the pipeline: https://github.com/cloud-bulldozer/scale-ci-pipeline/blob/master/jjb/dynamic/scale-ci-pipeline-bm-4.7.yml#L24? This way we can avoid running the benchmarks not intended to run at the node count desired. Thoughts?

Reply:
@smalleni Really I don't have any reason to skip this `multi-stage` scaling, lets make use of this existing workflow and include BM scaleup job at all stages and default it to `false` for baremetal as well. any thoughts ?

Reply:
Cool cool, lets add it in the conditionals but skip it in pipeline, sounds good

Reply:
@chaitanyaenr I removed it from pipeline file and added conditions in groovy let me know.